src/reporter/delete.py

```diff
@@ -34,11 +34,19 @@ def delete_entity(entity_id, type_=None, from_date=None, to_date=None):
 def delete_entities(entity_type, from_date=None, to_date=None,
                     drop_table=False):
     with translator_for(fiware_s()) as trans:
-        if drop_table:
-            trans.drop_table(etype=entity_type, fiware_service=fiware_s())
-            logging.getLogger(__name__).info(
-                "dropped entity_type {}".format(entity_type))
-            return 'entity table dropped', 204
+        if fiware_sp() is not None:
```

Comment:
remove from the pr the related commit.

Reply:
> remove from the pr the related commit.

@chicco785 , This commits is in master branch of my repository so once this [PR](https://github.com/orchestracities/ngsi-timeseries-api/pull/455) and all previous PR will get close it will not reflect in branch commit.

Reply:
but this commit needs changes.
in general it is always better 1 change - 1 pr

Reply:
can you add me as editor to your repo? gonna be faster

Reply:
> can you add me as editor to your repo? gonna be faster

Done. Thank you